manifests/server/config/factsource/yaml.pp

```diff
@@ -7,6 +7,9 @@
   $excluded_facts      = $mcollective::excluded_facts
   $yaml_fact_path_real = $mcollective::yaml_fact_path_real
 
+  $cron_minute_value   = fqdn_rand(60, ${::macaddress})
+  $cron_hour_value     = '0-23/2'
```

Comment:
I think this assumes a lot about the cron capabilities of the host. What if we did the below? Then fired on every hour.

$cron_minute_offset = fqdn_rand(15, ${::macaddress})
$cron_minutes = [ $cron_minute_offset, $cron_minute_offset + 15, $cron_minute_offset + 30, $cron_minute_offset + 45 ]